locales/ku/LC_MESSAGES/duckduckgo.po

```diff
@@ -24,12 +24,12 @@ msgstr "!Bang kurteyên lêgerînê"
 #. Heading for the table column indicating what percent of the population is fully vaccinated
 msgctxt "Covid 19 module"
 msgid "% Fully Vaccinated"
-msgstr "% 100% Derzî kirî"
+msgstr "% 100 Deqandî"
```

Comment:
Can you guys please double check these two strings? They don't seem to match the expectations when using a machine translator.

Reply:
Fixed :)

Reply:
The machine translation still returns weird translations like "detached" or "decayed", but if you guys are sure the translation is correct I'm happy to proceed.

Reply:
> The machine translation still returns weird translations like "detached" or "decayed", but if you guys are sure the translation is correct I'm happy to proceed.

@GioSensation Hi, I can see the weird results from different online translation services like  
Google/Bing/Lingvanex. I'll report it btw for a better experience.
Whatever you can check **Deqandî** meanings in[Wikiferheng (**Kurdish version of** **Wiki Dictionary**)](https://ku.m.wiktionary.org/wiki/deqand%C3%AE).
Thank you for mentioning that.